ta4j-core/src/main/java/org/ta4j/core/num/PrecisionNum.java

```diff
@@ -136,7 +136,7 @@ private PrecisionNum(BigDecimal val, int precision) {
      * @return the {@code Num}
      */
     public static PrecisionNum valueOf(String val) {
-        if (val.toUpperCase().equals("NAN")) {
```

Comment:
Why do you want to change this? `toUpperCase `+ `equals` is not clearer than just one method call `equalsIgnoreCase`. Also the performance of `equalsIgnoreCase` is a little bit better.

Reply:
? `val.toUpperCase().equals` was the code **before**..I wanted to replace to `equalsIgnoreCase`..